catkin_tools/commands/catkin.py

```diff
@@ -29,16 +32,25 @@
 
 CATKIN_COMMAND_VERB_GROUP = 'catkin_tools.commands.catkin.verbs'
 
+ENTRY_POINTS = None
+
+
+def _get_entry_points():
+    global ENTRY_POINTS
+    if ENTRY_POINTS is None:
+        ENTRY_POINTS = entry_points()
+    return ENTRY_POINTS
+
 
 def list_verbs():
     verbs = []
-    for entry_point in pkg_resources.iter_entry_points(group=CATKIN_COMMAND_VERB_GROUP):
+    for entry_point in _get_entry_points().get(CATKIN_COMMAND_VERB_GROUP, []):
```

Comment:
this is deprecated DeprecationWarning: SelectableGroups dict interface is deprecated. Use select.